packages/agent/src/agents/core/snakAgent.ts

```
@@ -3,10 +3,10 @@ import { RpcProvider } from 'starknet';
 import { ModelSelector } from '../operators/modelSelector.js';
 import {
   logger,
-  metrics,
   AgentConfig,
   CustomHuggingFaceEmbeddings,
 } from '@snakagent/core';
+import { metrics } from '@snakagent/metrics';
```

ESLint

[error] 'metrics' is defined but never used.

(@typescript-eslint/no-unused-vars)